### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,121 @@
...
+$ docker run --name sonarqube -v $PWD/qualityprofile:/qualityprofile -p 9000:9000 -p 9002:9002 --env-file ./env.file sonarqube
+```
+
+# Configuration
+
+You may add additional configuration which will brings you a read to run environment.
+
+## Quality Profile
+
+SonarQube default profiles may not feet what you are looking for.
```
Comment (mbaechler) [inline README.md]: s/feet/fit

**Comment by chamerling** [comment]
OMG JARs! Can't we rely on Maven instead? And you have a SNAPSHOT dependency...

File: qualityprofile/qualityprofile_java.txt [hunk 2]
```diff
@@ -0,0 +1 @@
...Key><key>S3355</key><priority>CRITICAL</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>S3398</key><priority>MAJOR</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>S3400</key><priority>MAJOR</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>S3421</key><priority>MAJOR</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>S3422</key><priority>MAJOR</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>S3438</key><priority>CRITICAL</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>S888</key><priority>CRITICAL</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>SwitchLastCaseIsDefaultCheck</key><priority>MAJOR</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>UnusedPrivateMethod</key><priority>MAJOR</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>UselessImportCheck</key><priority>MINOR</priority><parameters/></rule><rule><repositoryKey>squid</repositoryKey><key>UselessParenthesesCheck</key><priority>MAJOR</priority><parameters/></rule></rules></profile>
```
Comment (mbaechler) [inline qualityprofile/qualityprofile_java.txt]: xml indent ?

File: start_with_profile.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+# Start Sonar
+./bin/run.sh &
+
+# Wait for server to be up
+sleep 60
+
+# Restore qualityprofile
+if [ "$LANGUAGE" ] && [ "$PROFILE_NAME" ]; then
+    curl -v -u admin:admin -F "backup=@/qualityprofile/qualityprofile.txt" -X POST 'http://127.0.0.1:9000/api/qualityprofiles/restore'
```
Comment (mbaechler) [inline start_with_profile.sh]: try being DRY

File: README.md (hunk 1 above)
Comment (aduprat) [inline README.md]: OK

File: qualityprofile/qualityprofile_java.txt (hunk 2 above)
Comment (aduprat) [inline qualityprofile/qualityprofile_java.txt]: OK

File: start_with_profile.sh (hunk 3 above)
Comment (aduprat) [inline start_with_profile.sh]: OK

**Comment by aduprat** [comment]
Actually, only the GitHub plugin is available for download in:
https://sonarsource.bintray.com/Distribution/

The Stash plugin is not available, you have to compile it.
The sonar-stash-plugin.1.0.0 release was not working during my tests.

**Comment by chamerling** [comment]
:+1:

**Comment by Eriflow** [comment]
:+1:

**Comment by rpignolet** [comment]
:+1:

**Pull request merged**